design/FY2019/Heat Emissions from Buildings to Ambient.md

```diff
@@ -0,0 +1,216 @@
+
+This part of zone exhaust only includes the exhaust heat at the zone level (defined by _Fan:ZoneExhaust_ or _AirflowNetwork:MultiZone:Component:ZoneExhaustFan_). The exhaust heat from Zone HVAC system nodes is included in the next item. Besides, other part of air exchange with outdoors at system node level, balanced along with AirLoopHVAC, is considered in the next item.
+
+**3.	Heat emission from HVAC systems through relief air**
+
+HVAC systems relieve heat from system outdoor air relief nodes, and the total and sensible emission rate are calculated as:
+
+
+Q<sub>exh, sys</sub> = m<sub>exh</sub>(h<sub>zone</sub> - h<sub>out</sub>) &nbsp;&nbsp; Eq. (6),
+
+Q<sub>exh, sys, sensible</sub> = m<sub>exh</sub> * c * (T<sub>zone</sub> - T<sub>out</sub>) &nbsp;&nbsp; Eq. (7).
+
+And the latent parts are determined by taking the difference between the total and sensible rate.
+
+The air exchange with outdoor is represented with an _OutdoorAir:Mixer_ object in the  _AirLoopHVAC:OutdoorAirSystem_ object. 
+
+**4.	Heat rejection from HVAC system equipment**
+
+HVAC system equipment also directly exchanges heat with ambient air via refrigeration cycle and its energy usage. For example, for gas-fired boilers/heaters, waste heat is exhausted to outside together with combustion byproduct. For space cooling, heat is removed from indoors and dumped to outdoor at the condenser side or corresponding cooling equipment, such as cooling tower. In addition, energy consumed by outdoor equipment will directly dissipate to the ambient air, such as compressors and fans in packaged DX systems. In general, we assume the heat rejection is due to gas-powered combustion and condensers of any type, and we calculated these items based on the system type:
+
+(1)	Gas-powered combustion unit: Fuel generated heat - fuel heat supply
+
+(2)	Condensing unit:
+
+a.	Air-cooled: cooling rate + electric power of condenser fan and compressor
+
+b.	Water-cooled: total heat transfer rate with outdoor air
+
+Detailed calculation of HVAC rejected heat varies with different HVAC object groups and component types, as summarized in Table 1.
+
+**Table 1. Heat rejection calculation for different HVAC component types**
+
+Object Group | Component | Calculation Methods
+-------------  | -------------  | -------------
+Coils | Chilled water cooling coil | Emit heat in condensing unit defined in Condenser Equipment
+ |  | Hot water heating coil          | Emit heat in plant source defined in Plant Heating and Cooling Equipment
+ |  | DX cooling coil (air-cooled)    | Cooling Coil Source Side Heat Transfer Rate + Cooling Coil Electric Power
```

Comment:
Coil:Cooling:DX:MultiSpeed allows multiple fuel types, in addition to electricity. If the fuel type is not electricity, it will have waste heat. Do you include?

Reply:
@lgu1234 @mjwitte We would add the cases for other fuel types and the case for evaporatively cooled